README.md

```diff
@@ -46,7 +48,8 @@ This extension adds following methods to `Nette\Forms\Container` and thus to all
 
 ```php
 [
-	AjaxSelectExtension::CONFIG_INVALID_VALUE_MODE => AjaxSelectExtension::INVALID_VALUE_MODE_*,
+    AjaxSelectExtension::CONFIG_INVALID_VALUE_MODE => AjaxSelectExtension::INVALID_VALUE_MODE_*,
+    AjaxSelectExtension::CONFIG_DISABLE_INCLUDE_BY_ID_FILTER => FALSE,
```

Comment:
Pojmenujem to kladně jako `CONFIG_ENABLE_INCLUDE_BY_ID_FILTER` a dejme to defaultně true.